res/database/Default/user/rumpelfreddy.xml

```diff
@@ -703,612 +703,640 @@
+			</description>
+			<staff>
+				<member index="0" function="1">Freddy-realperson-ThomasUnterlisl</member>
 				<member index="1" function="2">Freddy-realperson-wilfredelster</member>
-            </staff>
-            <groups target_groups="0" />
-            <data country="D" year="2008" distribution="0" maingenre="12" subgenre="5" flags="68" blocks="2" price_mod="1.0" />
-            <ratings critics="50" speed="55" outcome="10" />
-        </programme>
+			</staff>
+			<groups target_groups="0" />
+			<data country="D" year="2008" distribution="0" maingenre="12" subgenre="5" flags="68" blocks="2" price_mod="1.0" />
+			<ratings critics="50" speed="55" outcome="10" />
+		</programme>
 		
-		 <programme id="Freddy_Film_TO_2" product="1" licence_type="1" fictional="0" creator="8667" created_by="Rumpelfreddy">
-            <title>
+		 <programme id="Freddy_Film_TO_2" product="1" licence_type="1" fictional="0" imdb_id="tt0806143" creator="8667" created_by="Rumpelfreddy">
+			<title>
 				<!-- Org. Operation Rienzi 2005 Kurzfim -->
-                <de>Codename Rienzi</de>
-                <en></en>
-            </title>
-            <description>
-                <de>Zeigt den schwierigen Weg des Frühstückseis unseres Präsidenten, vom Premiumhuhn auf den Teller.</de>
-                <en>Shows the difficult route of our president's breakfast egg. From vip chicken right on to their plate.</en>
-            </description>
-            <staff>
-                <member index="0" function="1">Freddy-realperson-ThomasUnterlisl</member>
+				<de>Codename Rienzi</de>
+				<en>Codename Rienzi</en>
+			</title>
+			<title_original>
+				<de>Operation Rienzi</de>
+				<en>Operation Rienzi</en>
+			</title_original>
+			<description>
+				<de>Zeigt den schwierigen Weg des Frühstückseis unseres Präsidenten, vom Premiumhuhn auf den Teller.</de>
+				<en>Shows the difficult journey of our president's breakfast egg, from the premium chicken to the plate.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">Freddy-realperson-ThomasUnterlisl</member>
 				<member index="1" function="2">Freddy-realperson-wilfredelster</member>
-            </staff>
-            <groups target_groups="0" />
-            <data country="D" year="2005" distribution="0" maingenre="5" subgenre="" flags="4" blocks="1" price_mod="1.0" />
-            <ratings critics="62" speed="24" outcome="12" />
-        </programme>
+			</staff>
+			<groups target_groups="0" />
+			<data country="D" year="2005" distribution="0" maingenre="5" subgenre="" flags="4" blocks="1" price_mod="1.0" />
+			<ratings critics="62" speed="24" outcome="12" />
+		</programme>
 		
-		 <programme id="Freddy_Film_TO_3" product="1" licence_type="1" fictional="0" creator="8667" created_by="Rumpelfreddy">
-            <title>
+		 <programme id="Freddy_Film_TO_3" product="1" licence_type="1" fictional="0" imdb_id="tt0349458" creator="8667" created_by="Rumpelfreddy">
```

Comment:
If I see it correctly, most changes in this file were fixing indentation.
This entry starts with tabs plus one space. Could you run a quick check if there are further cases (search in my browser not too helpful)?